doc/develop/getting_started/index.rst

```diff
@@ -533,6 +533,11 @@ that are used to emulate, flash and debug Zephyr applications.
             sudo cp ~/zephyr-sdk-0.14.2/sysroots/x86_64-pokysdk-linux/usr/share/openocd/contrib/60-openocd.rules /etc/udev/rules.d
             sudo udevadm control --reload
 
+      .. note::
+         Make sure the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` and :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables
+         are set correctly (to ``zephyr`` and your Zephyr SDK installation directory, respectively).
+         See the :ref:`toolchain_zephyr_sdk_bundle_variables` section for details.
+
```

Comment:
Nothing wrong with this line, but please refer to the commit (0a758c36c864cc138941848f42c8ea4aa7dc95b3) instead of the pull request number (PR#46319) in the commit log.

Git is a decentralized version control system. Commits are "forever" and "global" objects across forks. Pull request numbers are specific to individual forks. So the log will be clearer if you use the commit instead. This also allows people to figure out the changes you are talking about even when they are offline.

Reply:
This is now applied, thanks!

Reply:
@greg-fer the commit log says:

```
Edited the changes added in commit #0a758c3.
```

it should say:

```
Edited the changes added in commit 0a758c3.
```

Commit SHAs are not numbers :).